src/Athens-Examples/AthensTextRenderManualTests.class.st

```diff
@@ -224,17 +227,17 @@ AthensTextRenderManualTests class >> testWindow [
 AthensTextRenderManualTests class >> testWindowRender [
 	<example>
 	
-	| surf |
-	surf := self surfaceClass extent: Display extent.
+	| surf win |
+	win := self testWindow.
+	surf := self surfaceClass extent: win extent.
+	win position: 0@0.
 
 	surf drawDuring: [ :canvas | 
-		
 		canvas pathTransform scaleBy: 1.
-		canvas fullDrawMorph: self testWindow.
-
+		canvas fullDrawMorph: win.
 	].
 
-	Display getCanvas translucentImage: surf asForm at: 0@0
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?